### PR TITLE
Apicurio blogpost

### PR DESCRIPTION
**Pull request by jpechane**
Needs https://github.com/debezium/debezium-examples/pull/103

### CODE REVIEW

File: blog/2020-04-09-debezium-apicurio-registry.adoc [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+
+[source,json]
+----
+...
+"key.converter": "io.apicurio.registry.utils.converter.AvroConverter", <1>
+"key.converter.apicurio.registry.url": "http://apicurio:8080", <2>
+"key.converter.apicurio.registry.converter.serializer":
+    "io.apicurio.registry.utils.serde.AvroKafkaSerializer", <3>
+"key.converter.apicurio.registry.converter.deserializer":
+    "io.apicurio.registry.utils.serde.AvroKafkaDeserializer", <3>
```
Comment (gunnarmorling) [inline blog/2020-04-09-debezium-apicurio-registry.adoc:590]: Does it actually need the *de*serializer configuration for a source connector? Same for value below.

Comment (jpechane) [reply blog/2020-04-09-debezium-apicurio-registry.adoc:590]: It needs as they are instantiated in `configure` method, not lazily when/if needed.

Comment (gunnarmorling) [reply blog/2020-04-09-debezium-apicurio-registry.adoc:540]: Ok, gotcha. Why is it that the converter needs an explicit serializer config anyways? I.e. which (de)serializer should the `AvroConverter` use, if not the Avro ones?

File: blog/2020-04-09-debezium-apicurio-registry.adoc [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+        "transforms.key.field": "id",
+        "key.ignore": "false",
+        "type.name": "customer",
+        "behavior.on.null.values": "delete",
+        "key.converter": "io.apicurio.registry.utils.converter.AvroConverter",
+        "key.converter.apicurio.registry.url": "http://apicurio:8080",
+        "key.converter.apicurio.registry.converter.serializer": "io.apicurio.registry.utils.serde.AvroKafkaSerializer",
+        "key.converter.apicurio.registry.converter.deserializer": "io.apicurio.registry.utils.serde.AvroKafkaDeserializer",
+        "key.converter.apicurio.registry.global-id": "io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy",
+        "key.converter.apicurio.registry.id-handler": "io.apicurio.registry.utils.serde.strategy.ConfluentIdHandler",
```
Comment (gunnarmorling) [inline blog/2020-04-09-debezium-apicurio-registry.adoc:583]: Is this needed actually? Same below.

File: blog/2020-04-09-debezium-apicurio-registry.adoc (hunk 1 above)
Comment (jpechane) [reply blog/2020-04-09-debezium-apicurio-registry.adoc:540]: It is pluggable so you can use alternative serde implementations if you wish.
Unfortunately there is no sensible default set

File: blog/2020-04-09-debezium-apicurio-registry.adoc [hunk 3]
```diff
@@ -0,0 +1,616 @@
...
+        "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState",
+        "transforms.unwrap.drop.tombstones": "false",
+        "transforms.key.type": "org.apache.kafka.connect.transforms.ExtractField$Key",
+        "transforms.key.field": "id",
+        "key.ignore": "false",
+        "type.name": "customer",
+        "behavior.on.null.values": "delete",
+        "key.converter": "io.apicurio.registry.utils.converter.AvroConverter",
+        "key.converter.apicurio.registry.url": "http://apicurio:8080",
+        "key.converter.apicurio.registry.converter.serializer": "io.apicurio.registry.utils.serde.AvroKafkaSerializer",
```
Comment (gunnarmorling) [inline blog/2020-04-09-debezium-apicurio-registry.adoc:580]: Can you wrap and indent the very long lines as I did in other examples?

File: blog/2020-04-09-debezium-apicurio-registry.adoc (hunk 2 above)
Comment (jpechane) [reply blog/2020-04-09-debezium-apicurio-registry.adoc]: copy/paste error - removed

File: blog/2020-04-09-debezium-apicurio-registry.adoc (hunk 3 above)
Comment (jpechane) [reply blog/2020-04-09-debezium-apicurio-registry.adoc]: Done

**Comment by jpechane** [comment]
@gunnarmorling All should be added

File: blog/2020-04-09-debezium-apicurio-registry.adoc (hunk 1 above)
Comment (gunnarmorling) [reply blog/2020-04-09-debezium-apicurio-registry.adoc:548]: Ok; logged an issue with Apicurio: https://github.com/Apicurio/apicurio-registry/issues/378.

**Comment by gunnarmorling** [comment]
Sweet; pushed one more commit with a last few adjustments. Thanks a lot, nice write-up!

**Pull request merged**